lib/prefixed_ids/prefix_id.rb

```diff
@@ -13,7 +13,10 @@ def initialize(model, prefix, salt: PrefixedIds.salt, minimum_length: PrefixedId
     def encode(id)
       return if id.nil?
 
-      @prefix + @delimiter + @hashids.encode(TOKEN, id)
+      args = [TOKEN]
+      args += id.is_a?(Array) ? id : [id]
```

Comment:
I think this can be simplified a lot by using 

```suggestion
      Array.wrap(id)
```

Reply:
Yeah, agreed, I've made that change.

I also made a change to make the value the decode method clearer, standardrb was complaining about an unnecessary variable assignment which then made the tests fail when removed.